README.md

```diff
@@ -2,60 +2,106 @@
+| [@turnkey/sdk-browser](/packages/sdk-browser)                                                                        | The `@turnkey/sdk-browser` package exposes functionality that lets developers build browser based applications that interact with the Turnkey API with different types of authentication. It consists of different clients that enable requests to the API to be authenticated via different auth methods like user sessions, passkeys and iFrames. It also contains methods to manage information and state related to authentication like auth bundles and sessions, retrieving user information and server signing API requests.                                                                                                                                                                                                                                                    | [![npm](https://img.shields.io/npm/v/@turnkey/http?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/sdk-browser)                                          | [CHANGELOG](/packages/sdk-browser/CHANGELOG.md)                  | [Docs](https://docs.turnkey.com/sdks/javascript-browser) |
+| [@turnkey/sdk-server](https://github.com/tkhq/sdk/tree/main/packages/sdk-server)                                     | The `@turnkey/sdk-server` package exposes functionality that lets developers build server-side functionality for applications that interact with the Turnkey API with different types of authentication – allowing applications to authenticate users, manage sessions, and perform organizational operations securely and efficiently. It consists of an API Client and API Proxies that enable requests to the Turnkey API to be authenticated with the appropriate credentials. Specifically, the API Client manages requests signed by the user's authentication details, and the API proxies handle requests signed by the parent organization's authentication details. Use this package to handle server-side interactions for applications that interact with the Turnkey API. | [![npm](https://img.shields.io/npm/v/@turnkey/sdk-server?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/sdk-server)                                     | [CHANGELOG](/packages/sdk-server/CHANGELOG.md)                   | [Docs](https://docs.turnkey.com/sdks/javascript-server)  |
```

Comment:
> The `@turnkey/sdk-server` package exposes functionality that lets developers build server-side functionality for applications that interact with the Turnkey API with different types of authentication – allowing applications to authenticate users, manage sessions, and perform organizational operations securely and efficiently. It consists of an API Client and API Proxies that enable requests to the Turnkey API to be authenticated with the appropriate credentials. Specifically, the API Client manages requests signed by the user's authentication details, and the API proxies handle requests signed by the parent organization's authentication details. Use this package to handle server-side interactions for applications that interact with the Turnkey API.

The upshot of `@turnkey/sdk-server` is that it allows people to use Turnkey with an API key credential (P256 keypair that can be created using our `@turnkey/crypto` library)